### ISSUE TITLE
My Projects button doesn't scroll down to projects

### ISSUE THREAD

**Issue by tobiasrehm** [climateconnect/climateconnect#453]
"My Projects" button in personal header sub-menu just redirects to profile page without scrolling down to projects

**Comment by mdPial** [comment]
Hello, I would like to work on that issue 😃️

**Comment by positiveimpact** [comment]
@mdPial Great! I assigned it to you. Let me know if you have any questions or need any help setting up the project locally.

**Comment by positiveimpact** [comment]
You will need it to test locally. If you believe you can fix this without trying it out locally you won't need it. But I'd highly recommend it so you can make sure it actually works.
Since this is quite a simple change it could be possible without setting up the backend though.

**Comment by mdPial** [comment]
When I am running the front-end I am getting this error. I couldn't figure out.
![image](https://user-images.githubusercontent.com/43095489/105852137-4518e480-600e-11eb-81a2-9cf64ba33da4.png)

**Comment by positiveimpact** [comment]
The frontend can not work on its own with the current setup. 
You'll need to set up the backend to resolve this error and to be able to test with the frontend

**Comment by mdPial** [comment]
Okay. Thank you. Highly appreciated your quick response.

**Comment by mdPial** [comment]
Could you provide guideline in setting those environment variable? I am confused about the value.
API_HOST
API_URL
BASE_URL_HOST
DONATION_CAMPAIGN_RUNNING
ENVIRONMENT
GOOGLE_ANALYTICS_CODE
LATEST_NEWSLETTER_LINK
SOCKET_URL
LETS_ENCRYPT_FILE_CONTENT

**Comment by ddhanesha** [comment]
> Could you provide guideline in setting those environment variable? I am confused about the value.
> API_HOST
> API_URL
> BASE_URL_HOST
> DONATION_CAMPAIGN_RUNNING
> ENVIRONMENT
> GOOGLE_ANALYTICS_CODE
> LATEST_NEWSLETTER_LINK
> SOCKET_URL
> LETS_ENCRYPT_FILE_CONTENT

You **DO NOT** need these env variables...
1) LETS_ENCRYPT_FILE_CONTENT
2) DONATION_CAMPAIGN_RUNNING
3)GOOGLE_ANALYTICS_CODE
4) LATEST_NEWSLETTER_LINK

**Comment by mdPial** [comment]
> > Could you provide guideline in setting those environment variable? I am confused about the value.
> > API_HOST
> > API_URL
> > BASE_URL_HOST
> > DONATION_CAMPAIGN_RUNNING
> > ENVIRONMENT
> > GOOGLE_ANALYTICS_CODE
> > LATEST_NEWSLETTER_LINK
> > SOCKET_URL
> > LETS_ENCRYPT_FILE_CONTENT
> 
> You **DO NOT** need these env variables...
> 
> 1. LETS_ENCRYPT_FILE_CONTENT
> 2. DONATION_CAMPAIGN_RUNNING
> 3. GOOGLE_ANALYTICS_CODE
> 4. LATEST_NEWSLETTER_LINK

Could you give any example value of those?

**Comment by ddhanesha** [comment]
> > > Could you provide guideline in setting those environment variable? I am confused about the value.
> > > API_HOST
> > > API_URL
> > > BASE_URL_HOST
> > > DONATION_CAMPAIGN_RUNNING
> > > ENVIRONMENT
> > > GOOGLE_ANALYTICS_CODE
> > > LATEST_NEWSLETTER_LINK
> > > SOCKET_URL
> > > LETS_ENCRYPT_FILE_CONTENT
> > 
> > 
> > You **DO NOT** need these env variables...
> > 
> > 1. LETS_ENCRYPT_FILE_CONTENT
> > 2. DONATION_CAMPAIGN_RUNNING
> > 3. GOOGLE_ANALYTICS_CODE
> > 4. LATEST_NEWSLETTER_LINK
> 
> Could you give any example value of those?

Here is an example of env variables should be enough to test locally..
```
API_URL='http://127.0.0.1:8000'
ENVIRONMENT="develop"
SOCKET_URL="wss://127.0.0.1:8000"
BASE_URL_HOST = "localhost"
```

**Comment by mdPial** [comment]
Thank you @ddhanesha 😃️

**Comment by positiveimpact** [comment]
Resolved by @mdPial great work in #457

**Comment by mdPial** [comment]
> Resolved by @mdPial great work in #457

Thank You😃
